packages/app/src/components/RouteMap/RouteMap.js

```diff
@@ -41,6 +41,7 @@ const RouteMap = ({ restaurant }) => {
         tileSize={512}
         zoomOffset={-1}
         zoom={14}
+        style={{width:size+'px', height:size+'px'}}
```

Comment:
I think this should be in CSS? Inline styles are a bit iffy. Selectors could be used then to control the size in different contexts.